src/components/05_pages/NodeAddForm/index.js

```diff
@@ -17,30 +39,32 @@ class NodeAddForm extends React.Component {
 
   render() {
     return (
-      <Fragment>
-        <PageTitle>Create {this.props.bundle}</PageTitle>
-        <LoadingBar />
-        <NodeForm {...this.props} onSave={this.onSave} />
-      </Fragment>
+      this.props.schema && (
+        <Fragment>
+          <PageTitle>Create {this.props.bundle}</PageTitle>
+          <LoadingBar />
+          <NodeForm
+            {...this.props}
+            entity={createEntity(this.props.schema)}
```

Comment:
Should we add the requests for the default authored-by user and the created-on time to this?

Reply:
I guess that would make sense, wouldn't it?

Reply:
![](https://media.giphy.com/media/XdzAYATLJjSNO/giphy.gif)